components/crud-web-apps/jupyter/backend/apps/common/form.py

```diff
@@ -124,7 +124,7 @@ def set_notebook_cpu(notebook, body, defaults):
     if cpu_limit and 'nan' in cpu_limit.lower():
         raise BadRequest("Invalid value for cpu limit: %s" % cpu_limit)
 
-    limit_factor = utils.load_spawner_ui_config()["cpu"].get("limitFactor")
+    limit_factor = defaults["cpu"].get("limitFactor")
```

Comment:
Which version of Python is used? From Python 3.8 we could use walrus. It seems like an excellent place for it here.

Reply:
It's still 3.7: https://github.com/kubeflow/kubeflow/tree/master/components/crud-web-apps/jupyter#development
Would love to see a bump to 3.8 :)